docs/software/media.md

```diff
@@ -1124,4 +1126,123 @@ A FOSS web interface media streaming server, including live TV, forked from Emby
+
+The snapcast serve needs to have its audio sources manually configured after installation. See the snapcast documents for more details: <https://github.com/badaix/snapcast#server>
+
+TODO: Add DietPi specific setup e.g. to connect to raspotify, airport-sync and MPD.
+
+=== "Changing options"
+
+    - `/etc/snapserver.conf`
+
+    Then restart services with:
+
+    ```sh
+    systemctl restart snapserver
+    ```
+
+=== "Using MPD as an input source"
+    
+    If you have MPD installed you can use it as an input source.
+    
+    First you need to ensure that mpd outputs to a pipe you can do this in two ways.
+    1) Install CAVA from the DietPi software list. 
+    OR
+    2) Add the following to `/etc/mpd.conf` and restart with `systemctl restart mpd`
+
+    ```sh
+    audio_output {
+        type "fifo"
+        enabled "yes"
+        name "snapcast"
+        path "/tmp/mpd.fifo"
+        format "48000:16:2"
+    }    
+    ```
+    
+    Once you have done that you then need to add the following to `/etc/snapserver.conf` under `[stream]`. The `name` is the name as it will appear to snapcast clients here i've called it MyMpd. Check the snapserver docs for additional parameters you can pass in: <https://github.com/badaix/snapcast/blob/master/doc/configuration.md#pipe>
+
+    ```sh
+    source= pipe:///tmp/mpd.fifo?name=MyMpd&mode=read
+    ```
+
+=== "Using mopidy as an input source"
+    
+    If you have mopidy installed you can use it as an input source.
+    
+    First you need to ensure that mopidy outputs to a pipe. Add the following to `/etc/mopidy/mopidy.conf` and restart with `systemctl restart mopidy`
+
+    ```sh
+    [audio]
+    output = audioresample ! audioconvert ! audio/x-raw,rate=48000,channels=2,format=S16LE ! wavenc ! filesink location=/tmp/mopidy.fifo 
+    ```
+    
+    Once you have done that you then need to add the following to `/etc/snapserver.conf` under `[stream]`. The `name` is the name as it will appear to snapcast clients here i've called it MyMopidy. Check the snapserver docs for additional parameters you can pass in: <https://github.com/badaix/snapcast/blob/master/doc/configuration.md#pipe>
+
+    ```sh
+    source= pipe:///tmp/mopidy.fifo?name=MyMopidy&mode=read
+    ```
+
+=== "Using Raspotify/librespot as an input source"
+    
+    If you have raspotify installed you can use it as an input source.    
+
+    Add the following config under `[stream]`. The `name` is the name as it will appear to snapcast clients here i've called it MySpotify. The `devicename` is the name that will be shown when connecting in Spotify. Check the snapserver docs for additional parameters you can pass in: <https://github.com/badaix/snapcast/blob/master/doc/configuration.md#librespot>. I've disabled the audio cache to protect the SD card.
+
+    ```sh
+    source= librespot:///usr/bin/librespot?name=MySpotify&devicename=SnapcastSpotify&disable_audio_cache=true
+    ```
+
+=== "Using Airport (shareport-sync) as an input
```

Comment:
Isn't it actually called "AirPlay"? At least that is how Shairport Sync calls it. I don't use Apple devices, so 😄.

Reply:
Thanks. Fixed.